src/css/application.css

```diff
@@ -329,14 +329,32 @@ body {
   flex: 1 1 auto;
   display: flex;
   flex-direction: row;
+  height: 100vh;
```

Comment:
I’m curious to hear your reasoning here, but I would have been inclined to put this on the `environment-column` element rather than the `environment` block. No really firm reason for this, just that it makes more sense for elements to “push out” their block, rather than expanding to fill it.